purchases/src/main/kotlin/com/revenuecat/purchases/interfaces/ProductChangeCallback.kt

```diff
@@ -25,7 +25,7 @@ internal interface ProductChangeCallback : PurchaseErrorListener {
     fun onCompleted(paymentTransaction: PaymentTransaction?, customerInfo: CustomerInfo)
 }
 
-internal fun ProductChangeListener.toProductChangeCallback(): ProductChangeCallback {
+fun ProductChangeListener.toProductChangeCallback(): ProductChangeCallback {
```

Comment:
what's the plan for eventually deprecating these?

Reply:
I made these public in case folks want to do an easier migration, but were initially only used internally for not breaking people's compilation when deprecating `ProductChangeListener`. I think in 6 months to a year we can remove them. That's typically how long it takes us to remove deprecated functions.

Reply:
Also, I think I am going to try to make these not break because of the `PurchaserInfo` change either and do a conversion of the `PurchaserInfo` to `CustomerInfo`. Should be easy to do. I think that it will prevent devs from upgrading if everything breaks.

Reply:
ah, i see, good thought. i was wondering if we want to label THESE with a deprecated warning, since technically you're supposed to do warning before error and eventual removal. but i feel like that's kind a weird experience... thoughts?

also, could we just have the PurchaserInfo class use customerinfo underneath? then that class, at least, would have an automatic "toCustomerInfo"

Reply:
> i was wondering if we want to label THESE with a deprecated warning, since technically you're supposed to do warning before error and eventual removal. but i feel like that's kind a weird experience... thoughts?

I like it! The message can say something like this function will be removed in a future release, migrate to `ProductChangeCallback`

> also, could we just have the PurchaserInfo class use customerinfo underneath? then that class, at least, would have an automatic "toCustomerInfo"

That's exactly what I am thinking : heart_eyes_cat: